aws/resource_aws_ssm_patch_baseline.go

```diff
@@ -491,10 +518,18 @@ func flattenAwsSsmPatchRuleGroup(group *ssm.PatchRuleGroup) []map[string]interfa
 
 	for _, rule := range group.PatchRules {
 		r := make(map[string]interface{})
-		r["approve_after_days"] = aws.Int64Value(rule.ApproveAfterDays)
 		r["compliance_level"] = aws.StringValue(rule.ComplianceLevel)
 		r["enable_non_security"] = aws.BoolValue(rule.EnableNonSecurity)
 		r["patch_filter"] = flattenAwsSsmPatchFilterGroup(rule.PatchFilterGroup)
+
+		if rule.ApproveAfterDays != nil {
+      r["approve_after_days"] = aws.Int64Value(rule.ApproveAfterDays)
```

Comment:
This was my bad during rebase:

```suggestion
			r["approve_after_days"] = aws.Int64Value(rule.ApproveAfterDays)
```